samples/nrf9160/modem_shell/src/link/link.c

```diff
@@ -273,6 +273,21 @@ static int link_default_pdp_context_auth_set(void)
 	return 0;
 }
 
+static int link_enable_rel14_features(void)
+{
+	static const char rel14feat_at_cmd[] = "AT%REL14FEAT=1,1,1,1,1";
+	char response[MOSH_AT_CMD_RESPONSE_MAX_LEN + 1];
+
+	if (at_cmd_write(rel14feat_at_cmd, response,
```

Comment:
I'm not 100% sure whether this can happen here but 0 is a valid mobile equipment error code:
`CME ERROR: 0 -> Phone failure`
As said, not sure it can ever happen here. And I know we have dozens of places in nrf where failure is checked against return value 0 because our documentation is saying that 0 is a success, which it's not always. However, 100% correct way is to check the `state` parameter against `AT_CMD_OK`.